test/e2e/basic_scheduling/basic_scheduling_test.go

```diff
@@ -96,7 +96,6 @@ var _ = ginkgo.Describe("", func() {
 		gomega.Ω(allocations["partition"]).NotTo(gomega.BeNil())
 		gomega.Ω(allocations["uuid"]).NotTo(gomega.BeNil())
 		gomega.Ω(allocations["applicationId"]).To(gomega.Equal(sleepRespPod.ObjectMeta.Labels["applicationId"]))
-		gomega.Ω(allocations["queueName"]).To(gomega.ContainSubstring(sleepRespPod.ObjectMeta.Namespace))
```

Comment:
Is this related to the jira? or some other general cleanup?

Reply:
This is a follow up on YUNIKORN-1269 in which we removed the queueName from object and REST